<launch>
    <!-- ObjectDetection - TMR 2021 -->
    <env name="ROS_MASTER_URI" value="http://192.168.31.3:11311" />
    <env name="ROS_IP" value="192.168.31.5" />

    <!-- ObjectDetection -->
    <!-- VERBOSE: 'True' - Silence Mode, 'False' - Show images -->
    <arg name="VERBOSE" default="True"/>
    <node name="get_objects_and_coordinates" pkg="object_detection" type="get_objects_and_coordinates.py" respawn="true" output="screen" />
</launch>